src/session_impl.cpp

```diff
@@ -4357,6 +4357,19 @@ namespace aux {
 		m_stats_counters.set_value(counters::limiter_down_bytes
 			, m_download_rate.queued_bytes());
 
+		m_stats_counters.set_value(counters::total_download_rate
```

Comment:
the reason I haven't added these yet is because I can't really make up my mind about how to collate all these. Should every even write to all of (1) the peer connection's stats (2) the torrent's stats and (3) the session stats. Or should there be a wake-up each second (or so) to collect the accrued values from all of them.

I guess with the post_session_stats mechanism at least libtorrent doesn't _have_ to hard code some wake-up interval (even though, currently, the stats collection type does it once a second). I've been hesitating to extend the use of the stats type, because ideally the summing up of "child" stats should happen in here, and the stats type probably need to change to accommodate for that.

I suppose fixing that is orthogonal to this patch though.